luxus/style.css

```
@@ -0,0 +1,197 @@
+/*
+Theme Name: Luxus
+Theme URI: https://wordpress.com/theme/luxus
+Author: Automattic
+Author URI: https://automattic.com/
+Description: A WordPress theme for nail and beauty salons.
+Requires at least: 6.0
```

theme is using `fluid` typography (theme.json) which was introduced in [6.1](https://wordpress.org/news/2022/11/misha/).
```suggestion
Requires at least: 6.1
```